test/unit/lib/utils/telemetry/index.test.js

```diff
@@ -27,9 +27,9 @@ describe('analytics', () => {
   };
 
   before(() => {
-    process.env.SLS_ANALYTICS_URL = analyticsUrl;
-    ({ report, sendPending } = proxyquire('../../../../../lib/utils/analytics/index.js', {
-      '@serverless/utils/analytics-and-notfications-url': analyticsUrl,
+    process.env.SLS_ANALYTICS_URL = telemetryUrl;
```

Comment:
I think this env var can be removed (looks as same not cleaned leftover)

Reply:
Good catch, it's already mocked on next lines :+1: